.github/workflows/core-tests.yml

```diff
@@ -80,9 +80,9 @@ jobs:
         run: composer install -n
         working-directory: wordpress
 
-      - name: Downgrade PHPUnit
-        run: composer require --dev -n phpunit/phpunit:9.6.12
-        working-directory: wordpress
+      # - name: Downgrade PHPUnit
```

Comment:
Do we require the correct phpunit version by default now?

Reply:
I think so; 9.6.13 had some kind of a bug. The tests seem to work w/the latest version.